rest-backend/src/main/java/org/eclipse/steady/backend/repo/V_AppVulndepRepository.java

```diff
@@ -139,10 +139,10 @@
       value =
           " select count(*) from  ( select distinct"
               + " latest.mvn_group,a1.artifact,latest.latest_version from  (		select"
-              + " aa.mvn_group,aa.version as latest_version from app aa join 		(select app from ( "
-              + "			select distinct b.mvn_group,max(created_at) as max from 				(select distinct"
-              + " g.id,g.created_at,a.id,a.mvn_group from app_goal_exe g join app a on g.app=a.id"
-              + " ) as b 					group by b.mvn_group ) as c join app_goal_exe as l on"
+              + " aa.mvn_group,aa.version as latest_version from app aa join 		(select app from ( 	"
+              + "		select distinct b.mvn_group,max(created_at) as max from 				(select distinct"
+              + " g.id,g.created_at,a.id,a.mvn_group from app_goal_exe g join app a on g.app=a.id )"
+              + " as b 					group by b.mvn_group ) as c join app_goal_exe as l on"
```

Comment:
i think the auto-formatting messed up our queries. Anyways its not for this PR

Reply:
This were changes introduced by the new version of the google formatter. Do you think they pose problems?

Reply:
No, it should only affect humans :)